lib/org/chromium/webidl/PipelineMessage.js

```diff
@@ -0,0 +1,37 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
This change showed up in a dependent PR. Let's carry on the conversation about whether this is the right design there (or offline).